src/Selection.js

```diff
@@ -3,7 +3,7 @@ import closest from 'dom-helpers/query/closest'
 import events from 'dom-helpers/events'
 
 function addEventListener(type, handler, target = document) {
-  events.on(target, type, handler, { passive: false })
+  events.on(target, type, handler, { passive: false, capture: true })
```

Comment:
this is first prerequisite to fix the bug related to inability to resize events

Reply:
however, it's not sufficient yet - there is more work to be done here to make events / dnd working again ..